.github/workflows/create-jira-issues-for-community-activities.yml

```
@@ -46,10 +46,16 @@ on:
           If set to "true", Jira issue titles will be suffixed with "in <issue-component>" if the issue-component
           parameter is specified, or the repository name. If set to "false", no suffix will be added to Jira issue
           titles.
+      timeout-minutes:
+        required: false
+        type: number
+        default: 21
```

```suggestion
        default: 3
```
I feel 21 min a bit too much as the default here. See: https://github.com/Lombiq/Orchard-UIKit/actions/workflows/create-jira-issues-for-community-activities.yml

---

Fair point, 21 might be too high. It hits 4 minutes [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/workflows/create-jira-issues-for-community-activities.yml?page=4) and 14 minutes [here](https://github.com/Lombiq/GitHub-Actions/actions/workflows/create-jira-issues-for-community-activities-in-this-repo.yml?page=2). Also 3 minutes once in OSOCE. So maybe the default could be 5 or 6 minutes?

---

Sorry, I didn't get a notification about your comment. Yes, I can agree with 5min.